utils.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+import httpx
+
+class ModelName(str, Enum):
+    distilbert = "distilbert-base-cased"
+    roberta = "roberta-base-squad2"
+    google = "google-bert"
+    electra_large_discriminator = "electra_large_discriminator"
+
+API_URL = {
+    'distilbert-base-cased' : "https://api-inference.huggingface.co/models/distilbert/distilbert-base-cased-distilled-squad",
+    'roberta-base-squad2' : "https://api-inference.huggingface.co/models/deepset/roberta-base-squad2",
+    'google-bert' : "https://api-inference.huggingface.co/models/google-bert/bert-large-uncased-whole-word-masking-finetuned-squad",
+    'electra_large_discriminator': "https://api-inference.huggingface.co/models/ahotrod/electra_large_discriminator_squad2_512"
+}
+
+def selectModel(model: ModelName)->str:
+    return API_URL[model.value]
+
+async def send_rest_api_req(payload: dict, headers: dict, api_url: str)->dict:
+    async with httpx.AsyncClient() as client:
+        response = await client.post(headers= headers, url = api_url, json= payload)
+        data = response.json()
+        return data
```

Comment:
The functions in this class can be unit tested. Add some unit testing files. Every PR should come with either updates to unit tests or E2E tests. In this case E2E test can be defined via manual steps in the readme but there should be accompanying automated unit tests.

Reply:
the `send_rest_api_req` api is currently only calling hugging face api and returning the response received and no processing on data is being done. Should we write unit test for it? because as of now the unit test would only call the hugging face api

Reply:
Ok for this code as you are not post processing the data in any particular way from HuggingFace. Typically you may wish to do some exception handling/exception raising for when the rest call hits errors so that YOUR server can exit gracefully. 

You havent done that hence why no point in unit testing. For the stage of development we are at, that is fine. But a point to note for the future nonetheless.